Workbooks/WorkbooksMetadata.json

```diff
@@ -6647,7 +6647,26 @@
       "domains": [
         "IT Operations"
       ]
-}
+},
```

Comment:
A bracket "}" is missing above line 6650

Reply:
I gave you write access and fixed the bracket.